python311Packages.py-serializable: 0.12.0 -> 0.13.0

fabaff opened the pull request:
Diff: https://github.com/madpah/serializable/compare/refs/tags/v0.12.0...v0.13.0

Changelog: https://github.com/madpah/serializable/blob/0.13.0/CHANGELOG.md

## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff committed:
python311Packages.py-serializable: 0.12.0 -> 0.13.0

Diff: https://github.com/madpah/serializable/compare/refs/tags/v0.12.0...v0.13.0

Changelog: https://github.com/madpah/serializable/blob/0.13.0/CHANGELOG.md

alyssais commented:
@OfBorg eval

natsukium commented:
Result of `nixpkgs-review pr 258534` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>pip-audit</li>
    <li>pip-audit.dist</li>
    <li>python310Packages.cyclonedx-python-lib</li>
    <li>python310Packages.cyclonedx-python-lib.dist</li>
    <li>python311Packages.cyclonedx-python-lib</li>
    <li>python311Packages.cyclonedx-python-lib.dist</li>
  </ul>
</details>
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>python310Packages.py-serializable</li>
    <li>python310Packages.py-serializable.dist</li>
    <li>python311Packages.py-serializable</li>
    <li>python311Packages.py-serializable.dist</li>
  </ul>
</details>

fabaff committed:
python311Packages.py-serializable: 0.13.0 -> 0.14.0

Diff: https://github.com/madpah/serializable/compare/refs/tags/v0.13.0...0.14.0

Changelog: https://github.com/madpah/serializable/blob/0.14.0/CHANGELOG.md

fabaff committed:
python311Packages.py-serializable: 0.14.0 -> 0.15.0

Diff: https://github.com/madpah/serializable/compare/refs/tags/v0.14.0...v0.15.0

Changelog: https://github.com/madpah/serializable/blob/0.15.0/CHANGELOG.md

fabaff committed:
python311Packages.cyclonedx-python-lib: disable failing test

fabaff commented:
Result of `nixpkgs-review pr 258534` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages built:</summary>
  <ul>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>pip-audit</li>
    <li>pip-audit.dist</li>
    <li>python310Packages.cyclonedx-python-lib</li>
    <li>python310Packages.cyclonedx-python-lib.dist</li>
    <li>python310Packages.py-serializable</li>
    <li>python310Packages.py-serializable.dist</li>
    <li>python311Packages.cyclonedx-python-lib</li>
    <li>python311Packages.cyclonedx-python-lib.dist</li>
    <li>python311Packages.py-serializable</li>
    <li>python311Packages.py-serializable.dist</li>
  </ul>
</details>
